src/back-end/lib/db.ts

```diff
@@ -406,11 +417,20 @@ export async function approveAffiliation(connection: Connection, id: Id): Promis
 export async function deleteAffiliation(connection: Connection, id: Id): Promise<Affiliation> {
   const now = new Date();
   const [result] = await connection('affiliations')
-    .where({ id })
     .update({
       membershipStatus: MembershipStatus.Inactive,
       updatedAt: now
-    }, ['*']);
+    }, ['*'])
+    .where({
+      id
+    })
+    .whereIn('organization', function() {
+      this.select('id')
+          .from('organizations')
+          .where({
+            active: true
+          });
+    });
```

Comment:
Does the order of the `update` and `where*` calls matter?

Reply:
It doesn't no, Knex will construct the SQL the same way regardless.